apps/nsq_to_nsq/nsq_to_nsq.go

```diff
@@ -31,10 +31,8 @@ const (
 
 var (
 	showVersion = flag.Bool("version", false, "print version string")
-
-	topic       = flag.String("topic", "", "nsq topic")
 	channel     = flag.String("channel", "nsq_to_nsq", "nsq channel")
-	destTopic   = flag.String("destination-topic", "", "destination nsq topic")
+	singleDestinationTopic   = flag.String("single-destination-topic", "", "activate single destination nsq topic Note: default destination topic is the topic itself")
```

Comment:
@ploxiln @mreiferson  I am ok with either implementation, if we decide to keep it I can get rid of the rename of destTopic, I can also take out the single topic destination. 

Just note that the current multi topic support forces all destination tcp to accept all topic provided in the command line, I am not sure if this is ok with the original ask. It seems that user may want fine grain control of what nsqd accept what topic, but maybe not. I am open to discussion :)

I should also ask, what is testing criteria for pull request in this repository? I did some testing to make sure what I described in my commit is accurate, I am planning to do some load testing, is there any requirement I should be aware of?